dsp/src/trig.rs

```diff
@@ -143,4 +228,40 @@ mod tests {
         assert!(max_err.0 < 1.1e-5);
         assert!(max_err.1 < 1.1e-5);
     }
+
+    #[test]
+    fn atan2_absolute_error() {
```

Comment:
We need to make sure that we explicitly test the corner case inputs as well: min, max, -1, 0, 1.

Reply:
I'll add min and max. -1,0,1 are already included because `NUM_VALS` is odd. Would you like me to make this more clear with a comment, or something else?

Reply:
All good like that